### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by quavo19**
# in this milestone;
- I updated my animals' schema to add a primary key and an auto incrementor
- I added more queries and tables
- I inserted relevant data into each table and referenced each foreign key
## bellow is how to queries look like

Screenshot (117)

Screenshot (119)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -10,4 +10,24 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals
-ADD species TEXT;
\ No newline at end of file
+ADD species TEXT;
+
+CREATE TABLE owners (
+   id INT generated by default as identity,
+   full_name TEXT,
```
Comment (danmainah) [inline schema.sql]: - [ ] @quavo19 kindly change the datatype, it's best practise to use `varchar` to represent a string and not text,

File: schema.sql [hunk 2]
```diff
@@ -10,4 +10,24 @@ CREATE TABLE animals (
...
+   age INT,
+   PRIMARY KEY(id)
+);
+
+CREATE TABLE species (
+   id INT generated by default as identity,
+   name TEXT,
```
Comment (danmainah) [inline schema.sql]: - [ ] @quavo19 kindly change the datatype, it's best practise to use `varchar` to represent a string and not text,

File: queries.sql [hunk 3]
```diff
@@ -87,4 +87,38 @@ SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
...
+-- How many animals are there per species?
+SELECT species.name, COUNT(*) from species 
+left join animals on species.id = animals.species_id
+group by species.name;
+
+-- List all Digimon owned by Jennifer Orwell.
+SELECT * FROM owners 
+JOIN animals ON owners.id = animals.owner_id
+JOIN species ON species.id = animals.species_id
+WHERE owners.full_name = 'Jennifer Orwell';
```
Comment (danmainah) [inline queries.sql]: - [ ] @quavo19 great job on your query you forgot to add code to specify animal species being listed eg `WHERE owners.full_name = 'Dean Winchester' AND species.name = 'Digimon';`

`don't forget to update the screenshot query result to reflect the changes`

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @quavo19 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 3 above)
Comment (quavo19) [reply queries.sql]: @danmainah I don't think that's part 
there are seven queries and I wrote them all. for Dean Winchester, the query is at line 116

Comment (quavo19) [reply queries.sql]: @danmainah  please which part of the requirement requires this `owners.full_name = 'Dean Winchester' AND species.name = 'Digimon';`
image

Comment (danmainah) [reply queries.sql]: - [ ] @quavo19 you misunderstood that is just an example query, the requirement was you list the animal owned by Jennifer Orwell from the Digimon species, your query only showed animals owned by Jennifer Orwell.

**Review by Kamba56 — APPROVED** [review]
Hi @quavo19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏
- [x] Excellent communication with the reviewer
- [x] Great use of JOIN clause in querying the database

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**